autopush_rs/src/server/mod.rs

```diff
@@ -463,6 +463,20 @@ impl Server {
         Err("User not connected".into())
     }
 
+    /// A check for notification command has come for the uaid
+    pub fn check_client_storage(&self, uaid: Uuid) -> Result<()> {
+        let mut uaids = self.uaids.borrow_mut();
```

Comment:
I don't think you need mut here or below? (use .borrow() and .get())

Reply:
@alexcrichton why was a borrow_mut() used on the notify_client func I coppied instead of borrow/get?

Reply:
Hm I think maybe that was from a long time ago? It may have originally done something mutable but may have been refactored away!

Hm I think maybe that was from a long time ago? It may have originally done something mutable but may have been refactored away!